src/poker.js

```diff
@@ -1,5 +1,120 @@
+// input an array with two arrays [['K', '5'], ['3', '7']]
+// output is an array
+// if neither array has pair return empty array
+// if only one pair then that wins
+// if two arrays then map score and compare
+
 class Poker {
-  // Write your implementation here
+  constructor(array) {
+    this.example = [];
+    this.array = array;
+    this.scoreMap = {
```

Comment:
I should incorporate more objects in my code... it looks neater, rather than creating another method. Cool!